.gitignore

```diff
@@ -25,4 +25,6 @@ Manifest.toml
 
 .vscode/
 
-sandbox/
\ No newline at end of file
+sandbox/
+
+/.config.toml
```

Comment:
As noted, the config file really shouldn't be in the package directory but I guess I'm okay with it being specified in the gitignore file.

Reply:
Ah, you put it in sandbox
> Create the `.config.toml` file and save to the sandbox directory.

I put it in root out of habit from other languages. At least this will ignore it if anyone else does the same thing.